crates/fluss/src/client/table/remote_log.rs

```diff
@@ -115,11 +116,19 @@ impl RemoteLogDownloadFuture {
 /// Downloader for remote log segment files
 pub struct RemoteLogDownloader {
     local_log_dir: TempDir,
+    s3_props: RwLock<HashMap<String, String>>,
```

Comment:
In here, we hard code to s3 props, but what if the remote storage is not s3?

Reply:
we can still keep the logic, but not to name it `s3_props`. I suggest:

- rename `s3_props` in `RemoteLogDownloader` to `remote_fs_props`
- rename `set_s3_props` to `set_remote_fs_props`
- `to_s3_props` to `to_remote_fs_props`